string class_label
int64[4] box_coordinates
